keel-core/src/main/kotlin/com/netflix/spinnaker/keel/api/RegionSpec.kt

```diff
@@ -18,5 +18,17 @@
 package com.netflix.spinnaker.keel.api
 
 interface RegionSpec {
-  val region: String
+  val name: String
```

Comment:
Changed this because it makes the YAML and code representations make more sense.